chain/chain/src/runtime/mod.rs

```diff
@@ -828,11 +828,31 @@ impl RuntimeAdapter for NightshadeRuntime {
     }
 
     fn get_gc_stop_height(&self, block_hash: &CryptoHash) -> BlockHeight {
+        use std::sync::atomic::{AtomicI64, Ordering};
+        use std::time::{SystemTime, UNIX_EPOCH};
+
+        // Use AtomicI64 to store unix timestamp in seconds
+        static LAST_LOG_TIME: AtomicI64 = AtomicI64::new(0);
+        // Log throttling interval (10 seconds)
```

Comment:
I don't see a lot of additional information in these comments on top of what I can deduce from the variable names.  I propose removing them.